src/scancode/format.py

```diff
@@ -81,21 +77,26 @@ class HtmlAppAssetCopyError(Exception):
     pass
 
 
+def is_stdout(output_file):
+    return output_file.name == '<stdout>'
+
+
 def get_html_app_files_dirs(output_file):
     """
     Return a tuple of (parent_dir, dir_name) directory named after the
     `output_file` file object file_base_name (stripped from extension) and a
-    `_files` suffix Return None if output is to stdout.
+    `_files` suffix Return empty strings if output is to stdout.
     """
+    if is_stdout(output_file):
+        return '', ''
+
     file_name = output_file.name
-    if file_name == '<stdout>':
```

Comment:
@pombredanne The change was to make the check for '\<stdout>' more explicit. 

**BEFORE**:
<pre><code>
tgt_dirs = get_html_app_files_dirs(output_file)
if not tgt_dirs:
    raise HtmlAppAssetCopyWarning()         # not clear warning from stdout
</code></pre>
**AFTER**:

<pre><code>
if is_stdout(output_file):
    raise HtmlAppAssetCopyWarning()
tgt_dirs = get_html_app_files_dirs(output_file)         # output valid, no need to check
</code></pre>